.github/workflows/ci.yml

```
@@ -23,7 +23,9 @@ jobs:
     - run: make requirements
     - run: make test NPM_TESTS=build
     - run: make test NPM_TESTS=${{ matrix.npm-test }}
-    - name: upload coverage
-      uses: codecov/codecov-action@v3
+    - name: Coverage
+      if: matrix.npm-test == 'test'
```

```suggestion
      if: ${{ matrix.npm-test == 'test' }}
```

I think this is the syntax we need here (just going off of https://docs.github.com/en/actions/learn-github-actions/expressions

---

It seems our wiki was wrong, then.  Good catch.